components/gc-follow-us/followus.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @title WET-BOEW Follow us component
+ * @overview Plugin used to replace Twitter with "X" - Deprecated
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @garneauma
+ */
+( function( $, window, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "followus",
+	selector = "." + componentName,
```

Comment:
We might need to support the conversion of the other use of the twitter icon. Like on this working example page: https://wet-boew.github.io/GCWeb/templates/widgets-en.html

So, this plugin will need to support also situation where ".icon.twitter" is used where the text is fully visible.